### PR TITLE
LPD-19215 Use parameterized preparedStatements

### PR DESCRIPTION
**Pull request by lfbesada**
A followup for [LPD-19215](https://liferay.atlassian.net/browse/LPD-19215)
ref https://github.com/brianchandotcom/liferay-portal/pull/147835#issuecomment-1979097739

cc: @balazssk

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:forward

File: modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/v1_4_1/LayoutClassedModelUsageUpgradeProcess.java [hunk 1]
```diff
@@ -260,10 +261,11 @@ private boolean _hasVisibleAssetEntry(long classNameId, long classPK)
 		try (PreparedStatement preparedStatement = connection.prepareStatement(
 				SQLTransformer.transform(
```
Comment (balazssk) [inline modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/v1_4_1/LayoutClassedModelUsageUpgradeProcess.java:262]: hey @lfbesada ,
I think if you remove the [$TRUE$] token, you can also remove the SQLTransformer, as there won't be anything left to transform

Comment (lfbesada) [reply modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/v1_4_1/LayoutClassedModelUsageUpgradeProcess.java:262]: true! thanks a lot!

**Comment by lfbesada** [comment]
ci:stop

**Comment by lfbesada** [comment]
ci:report:2547592635

**Comment by lfbesada** [comment]
ci:forward:force

**Pull request closed without merging**